reladomo/src/main/java/com/gs/fw/common/mithra/attribute/NonPrimitiveAttribute.java

```diff
@@ -317,4 +317,6 @@ public Operation notIn(Set<Type> set)
 //    public abstract Operation notIn(Set<Type> set);
 
     public abstract void setSqlParameter(int index, PreparedStatement ps, Object o, TimeZone databaseTimeZone, DatabaseType databaseType) throws SQLException;
+
+    public abstract String formattedValue(Object object);

```

Comment:
why is this not generic to `Type`? Also, you can pull up the quoted implementation up here and only override in BigDecimal and ByteArray.

Reply:
Good point, I changed `Object` to `Type`